packages/server/src/common/migrations/1760649528955-InitialSetup.ts

```diff
@@ -24,12 +25,41 @@ export class InitialSetup1760649528955 implements MigrationInterface {
       ],
     });
 
+    const eventsTable = new Table({
+      name: this.EVENTS_TABLE,
+      columns: [
+        {
+          name: 'id',
+          type: 'uuid',
+          isPrimary: true,
+          isUnique: true,
+          isNullable: false,
+          isGenerated: true,
+          generationStrategy: 'uuid',
+        },
+        { name: 'title', type: 'varchar', isNullable: false },
+        { name: 'description', type: 'text', isNullable: false },
+        { name: 'date', type: 'timestamptz', isNullable: false },
+        { name: 'location', type: 'varchar', isNullable: false },
+        { name: 'zip_code', type: 'integer', isNullable: false },
+        { name: 'council_id', type: 'uuid', isNullable: false },
```

Comment:
This should be unique

```suggestion
        { name: 'council_id', type: 'uuid', isUnique: true, isNullable: false },
```

Reply:
`council_id` cannot be unique as multiple events can occur at the same city council.

Reply:
That makes sense.